packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { DocsContext } from '@storybook/addon-docs';
+import dedent from 'dedent';
+import Box from '~components/Box';
+
+export type SandboxProps = {
+  children: string;
+  language?: 'js' | 'ts' | 'jsx' | 'tsx';
+};
+
+function Sandbox({ children, language = 'tsx' }: SandboxProps): JSX.Element {
+  const {
+    // @ts-expect-error globals is available but the typings in storybook are properly defined hence, ignoring it
+    globals: { themeTokenName, colorScheme },
+  } = React.useContext(DocsContext);
+
+  return (
+    <Box>
+      <br />
+      <Sandpack
+        template="react-ts"
+        files={{
+          '/index.tsx': dedent`import { StrictMode } from "react";
+            import * as ReactDOMClient from "react-dom/client";
+            import styled, { createGlobalStyle } from "styled-components";
+  
+            import { BladeProvider, Theme } from "@razorpay/blade/components";
+            import { ${themeTokenName} } from "@razorpay/blade/tokens";
+            import "@fontsource/lato/400.css";
+            import "@fontsource/lato/700.css";
+            
+            import App from "./App";
+  
+            const BackgroundBox = styled.div(
+              ({ theme }: { theme: Theme }) => ({
+                backgroundColor: theme.colors.surface.background.level1.lowContrast,
+                height: '100vh',
+                width: '100vw',
+                padding: '12px'
+              })
+            );
+
+            const GlobalStyles = createGlobalStyle\`
+              body {
+                margin: 0;
+                padding: 0;
+              }
+            \`;
+            
+            const rootElement = document.getElementById("root");
+            
+            if (!rootElement) {
+              throw new Error("root is null");
+            }
+            
+            const root = ReactDOMClient.createRoot(rootElement);
+            
+            root.render(
+              <StrictMode>
+                <BladeProvider themeTokens={${themeTokenName}} colorScheme="${colorScheme}">
+                  <GlobalStyles />
+                  <BackgroundBox>
+                    <App />
```

Comment:
what if directly do `dedent(children),` directly here inline? it doesn't work?

Reply:
it does but then users have to edit this cluttered file to make any changes. With this separated out in `App` component, users get this separate simple `App.tsx`  
<img width="597" alt="image" src="https://user-images.githubusercontent.com/30949385/199652757-8ed7bcb9-d1b3-4a61-874b-6fb7b4d3f975.png">